README.md

```diff
@@ -199,108 +102,79 @@ $ sudo ./setup_host_rpi.sh
-
-### Linux
-
-#### Raspberry Pi Setup
-
-A Raspberry Pi is not required on Linux, unless you want to use Nocturne in your car.
-
-Download and open [Raspberry Pi Imager](https://downloads.raspberrypi.org/imager/imager_latest.dmg), select Raspberry Pi OS (Legacy, 64-bit) Lite, select "Edit Settings", check "Set hostname", check "Set username and password" (set a password), check "Configure wireless LAN", (enter your network's SSID and password), check "Set local settings". Open the Services tab, enable SSH, and use password authentication. Write the configured OS to your microSD card and insert it into your Raspberry Pi.
-
-#### Flashing Process
-
-If you haven't already, download [superbird-tool](https://github.com/thinglabsoss/superbird-tool) and run the setup process detailed [here](https://github.com/thinglabsoss/superbird-tool?tab=readme-ov-file#supported-platforms).
-
-Download and unzip the latest image from [Releases](https://github.com/brandonsaldan/nocturne-image/releases), connect Car Thing to your computer in USB Mode (hold preset buttons 1 and 4 while connecting), and run the following from your command line:
 
-```bash
-# Go into the superbird-tool repository
-$ cd /path/to/superbird-tool-main
+<br>
 
-# Find device
-$ sudo python3 ./superbird_tool.py --find_device
+<details>
+<summary>MacOS (Not Recommended)</summary>
+<br>
 
-# Flash Nocturne image, without resetting the data partition 
-$ sudo python3 ./superbird_tool.py --dont_reset --restore_device /path/to/nocturne-image/image
-```
-After the flashing completes, unplug and replug your Car Thing, and change directories to the scripts folder.
+TBD
 
-```bash
-# Go into the setup-scripts folder
-$ cd /path/to/nocturne-image/setup-scripts
-```
+</details>
 
-There are two different ways to use Nocturne. You can either use it at your desk, or in your car. 
+<br>
 
 <details>
-<summary>Using Nocturne at your desk</summary>
+<summary>Linux (Not Recommended)</summary>
 <br>
 
-At this point, there are two different scripts that you can use. The first one, `setup_host_apt.sh`, is used on Linux distros that utilize apt as it's package manager. The second one, `setup_host_pacman.sh`, is used on Linux distros that utilize Pacman as it's package manager. If you use Pacman, replace `setup_host_apt.sh` in the following commands with `setup_host_pacman.sh` to continue with setup.
-
-It is important to note that `setup_host_pacman.sh` has not been tested very well, so it may not work as intended for setup. 
+TBD
 
-``` bash
-# Make setup_host_apt.sh executable
-$ chmod +x setup_host_apt.sh
-
-# Execute setup_host_apt.sh
-$ sudo ./setup_host_apt.sh
-```
 </details>
 
+<br>
+
```

Comment:
Why add these if they aren't completed? These two sections on using Windows/macOS as a host device should be removed.